### DISCUSSION TITLE
Filter function error

### DISCUSSION THREAD

**Discussion by arip7608** [teamalgoritma/community#879] [Q&A] [answered]
Ketika menggunakan filter function menunjukkan error
![capture filter error](https://github.com/teamalgoritma/community/assets/173215925/86e03f73-d166-47d0-8bf7-1e9b0a3eea63)

**Comment by ramdhinafinita** [comment, marked as answer]
Dear @arip7608 error dikarenakan terdapat typo pada kata **month** seharusnya **Month**, hati-hati dengan nama kolom. Terimakasih, semoga membantu

**Reply by arip7608** [reply]
terima kasih
